### PR TITLE
added is_callable() param-out

### PR DESCRIPTION
**Pull request by staabm**
I have never used this parameter, but stills.. thats the last function with a by-ref param from the php top-used 100 functions list

### CODE REVIEW

File: stubs/core.stub [hunk 1]
```diff
@@ -296,3 +296,8 @@ function headers_sent(string &$filename = null, int &$line = null): bool {}
  * @param-out array<string, string|array<string>> $result
  */
 function mb_parse_str(string $string, &$result = []): bool {}
+
+/**
+ * @param-out callable|false $callable_name
```
Comment (ondrejmirtes) [inline stubs/core.stub]: Should rather be callable-string. And have you verified what's set in case of non-callable being passed in?

**Review by ondrejmirtes — CHANGES_REQUESTED** [review]

File: stubs/core.stub (hunk 1 above)
Comment (staabm) [reply stubs/core.stub]: yep, I did.

https://3v4l.org/2UpJB

Comment (ondrejmirtes) [reply stubs/core.stub]: So it's never false?

What about if you pass in a nonexistent method via the array syntax?

Comment (staabm) [reply stubs/core.stub]: Ahh right: it doesn't get false

https://3v4l.org/HctjS

**Comment by ondrejmirtes** [comment]
Thank you.

**Pull request merged**